externs/ngeox.js

```diff
@@ -327,6 +327,39 @@ ngeox.MeasureEvent.prototype.feature;
 
 
 /**
+ * Options for the mobile geolocations directive.
+ * @typedef {{
+ *    accuracyFeatureStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    positionFeatureStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    zoom: (number|undefined)
+ * }}
+ */
+ngeox.DesktopGeolocationDirectiveOptions;
```

Comment:
Excellent point.  I went that single type approach at first, then chose to have them separated even if they have the same options.  They are currently exactly the same.  I don't have a strong opinion on this matter.  Let me know which one you prefer.